src/internal/kopia/upload.go

```diff
@@ -186,20 +187,29 @@ func (cp *corsoProgress) FinishedFile(relativePath string, err error) {
 
 	parent := d.repoPath.ToBuilder().Dir()
 
+	var locationFolders string
+	if d.locationPath != nil {
+		locationFolders = d.locationPath.Folder()
```

Comment:
`Folder()` does not provide something that can be turned into a `path.Path` later on. It's really only the part after the prefix and it will continue to include things like the drive ID and `root:` for OneDrive paths if we don't make further changes

Not everything needs to be solved in this PR, but trying to give a sense in how it will play out in the long run

Reply:
Right, I think that's correct for what I'm aiming to implement here.  It should, if I'm not mistaken, match the value we get out of `locationRefOf(path)` in `details.go`.

Reply:
Just putting here for completeness, one of my comments in another PR mentioned that this actually needs to be the escaped version of the string (`Folder()` is not escaped, I don't remember why at this point :( ). Original [comment](https://github.com/alcionai/corso/pull/2429#discussion_r1099229056)